api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -182,6 +182,30 @@ message RedisProxy {
     Route catch_all_route = 4;
   }
 
+  // RedisFault defines faults used for fault injection.
+  message RedisFault {
+    enum RedisFaultType {
+      // Delays requests. This is the base fault; other faults can have delays added.
+      DELAY = 0;
+
+      // Returns errors on requests.
+      ERROR = 1;
+    }
+
+    // Fault type.
+    RedisFaultType fault_type = 1 [(validate.rules).enum = {defined_only: true}];
+
+    // Percentage of requests fault applies to.
```

Comment:
What is the default?

Reply:
This is required- I was under the impression required fields did not need a default.

Reply:
If it's required then you need a required validation.

Reply:
Does `[(validate.rules).message = {required: true}];` not do that?

Reply:
Sorry yes this is fine.